Firestore/core/src/core/view.h

```diff
@@ -196,8 +175,8 @@ class View {
    */
   ViewChange ApplyChanges(
       const core::ViewDocumentChanges& doc_changes,
-      const absl::optional<remote::TargetChange>& target_change,
-      bool targetIsPendingReset);
+      const absl::optional<remote::TargetChange>& target_change = {},
```

Comment:
nit: Please add `@param` tags for the new method parameters.

Reply:
`@param targetIsPendingReset` is added.

Reply:
Oh, right. My bad.